docs/fsharp/language-reference/computation-expressions.md

```diff
@@ -77,6 +78,29 @@ If you bind the call to a computation expression with `let`, you will not get th
 
 `let!` is defined by the `Bind(x, f)` member on the builder type.
 
+### `and!`
+
+The `and!` keyword binds the result of a call to another computation expression to a name, unlike `let!` it does not allow you to use previous results:
+
+```fsharp
+// OK
+let doThingsAsync url otherUrl =
+    async {
+        let! data = getDataAsync url
+        and! moreData = getDataAsync otherURL
```

Comment:
The problem here is that, by default, `async` doesn't have support for `and!`, and no built-in computation expression does.

I'm not sure what to do about this.  Perhaps just adjust the example to use one of the computation expressions here: https://github.com/dsyme/fsharp-presentations/blob/master/design-notes/rethinking-applicatives.md.  Then link to that as a resource at the end.  Anything more will require an extensive rewrite of the whole page I think.